tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -403,9 +403,11 @@ private void Run_NodeFails_ToInstallStretchSdk_OnNonStretchImage(string imageTag
         }
 
         [Theory, Trait("category", "githubactions")]
-        [Trait("build-image", "github-actions-debian-bullseye")]
+        [Trait("build-image", "github-actions-debian-bullseye-and-bookworm")]
```

Comment:
I think we should maintain the pattern we have right now of a job building and testing a single build image -- this will prevent confusion about which attribute to use for tests and minimize the amount of time spent in a single job building and testing a set of images rather than one image.

Reply:
that makes sense! I've moved the bookworm test out as a separate test in the latest commit. I also added the traits for runtime image integration tests to the yaml template for node runtime integration test.